src/OrchardCore.Themes/TheTheme/Views/Widget-Form.cshtml

```diff
@@ -0,0 +1,7 @@
+@{
+    Model.Classes.Add("widget-body mb-3");
```

Comment:
Why we need to add them into the model?

Reply:
The mb-3 is for look, and the widget-body is just to reduce the chance of breaking existing code. Some users may be using widget-body to override style. Widget-body is a class that is added when Widget.cshtml is used.